rosidl_runtime_py/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosidl_runtime_py</name>
+  <version>0.6.2</version>
+  <description>Runtime utilities for working with generated ROS interfaces in Python.</description>
+  <maintainer email="scott@openrobotics.org">Scott K Logan</maintainer>
```

Comment:
@jacobperron Is it intentionally that you create the package but put someone else as a maintainer into the manifest?

Reply:
No, this is a mistake.

Reply:
Fixed.